crypto/x509/v3_alt.c

```diff
@@ -121,7 +122,25 @@ STACK_OF(CONF_VALUE) *i2v_GENERAL_NAME(X509V3_EXT_METHOD *method,
                 return NULL;
             break;
         default:
-            if (!X509V3_add_value("othername", "<unsupported>", &ret))
+            if (OBJ_obj2txt(oline, sizeof(oline), gen->d.otherName->type_id, 0)) 
+                snprintf(othername, sizeof(othername), "othername: %s:", oline);
+            else
+                strncpy(othername, "othername", sizeof(othername));
```

Comment:
Why not `"othername:"`?

Reply:
well - because it looked nicer. But you are right - it is not identical to the old behaviour. fixed.